src/components/add-to-playlist-drawer/AddToPlaylistDrawer.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useCallback, useState } from 'react'
+
+import { CreatePlaylistSource } from 'audius-client/src/common/models/Analytics'
+import { getAccountWithOwnPlaylists } from 'audius-client/src/common/store/account/selectors'
+import {
+  addTrackToPlaylist,
+  createPlaylist
+} from 'audius-client/src/common/store/cache/collections/actions'
+import { close } from 'audius-client/src/common/store/ui/add-to-playlist/actions'
+import {
+  getIsOpen,
+  getTrackId,
+  getTrackTitle
+} from 'audius-client/src/common/store/ui/add-to-playlist/selectors'
+import { toast } from 'audius-client/src/common/store/ui/toast/slice'
+import { newCollectionMetadata } from 'audius-client/src/schemas'
+import { FEED_PAGE, playlistPage } from 'audius-client/src/utils/route'
+import {
+  NativeScrollEvent,
+  NativeSyntheticEvent,
+  ScrollView,
+  StyleSheet,
+  View
+} from 'react-native'
+import { Shadow } from 'react-native-shadow-2'
+
+import Button, { ButtonType } from 'app/components/button'
+import { Card } from 'app/components/card'
+import Drawer from 'app/components/drawer'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { usePushRouteWeb } from 'app/hooks/usePushRouteWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors } from 'app/utils/theme'
+
+const messages = {
+  title: 'Add To Playlist',
+  addedToast: 'Added To Playlist!',
+  createdToast: 'Playlist Created!'
+}
+
+const createStyles = (themeColors: ThemeColors) =>
```

Comment:
not a huge deal, but we have the `makeStyles` helper, which can comes with a spacing util and palette!